dotnet/src/Extensions/Planning.SequentialPlanner/SKContextSequentialPlannerExtensions.cs

```diff
@@ -33,7 +33,16 @@ public static class SKContextSequentialPlannerExtensions
         SequentialPlannerConfig? config = null)
     {
         config ??= new SequentialPlannerConfig();
-        var functions = await context.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false);
+
+        IOrderedEnumerable<FunctionView> functions;
+        if (config.GetAvailableFunctionsAsync is null)
+        {
+            functions = await context.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false);
+        }
+        else
+        {
+            functions = await config.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false);
+        }
```

Comment:
nit: A little simplification to reduce nesting :)
```suggestion
        IOrderedEnumerable<FunctionView> functions = config.GetAvailableFunctionsAsync is null ?
            await context.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false) :
            await config.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false);
```

Reply:
Yeah I was kind of puzzled when I saw this looking at my self review -- I think some IDE tooling re-did this away from your suggestion at some point in my development. Gonna try and go back cause I agree with you :)